src/scripts/main.js

```diff
@@ -1,3 +1,204 @@
 'use strict';
 
-// write code here
+function getNumber(str) {
+  return str.includes('$')
+    ? +str.slice(1).replaceAll(',', '')
+    : +str;
+}
+
+function replaceFirstLetter(str) {
+  return str.slice(0, 1).toUpperCase() + str.slice(1);
+}
+
+function pushNotification(type, text, titleText) {
+  const boxNotification = document.createElement('div');
+  const h1 = document.createElement('h1');
+  const p = document.createElement('p');
+
+  boxNotification.append(h1, p);
+  boxNotification.classList.add('notification', type);
+  h1.className = 'title';
+  h1.innerText = titleText;
+  p.innerText = text;
+  document.body.append(boxNotification);
+
+  setTimeout(() => {
+    boxNotification.remove();
+  }, 3000);
+}
+
+function getThousandSeparator(number) {
+  return number.toLocaleString('en-Us');
+}
+
+const data = ['name', 'position', 'office', 'age', 'salary'];
+const offices = ['Tokyo',
+  'Singapore', 'London', 'New York', 'Edinburgh', 'San Francisco'];
+const employees = document.querySelector('tbody');
+const form = document.createElement('form');
+const button = document.createElement('button');
+
+button.innerText = 'Save to table';
+
+form.className = 'new-employee-form';
+
+document.querySelector('body').append(form);
+
+for (let i = 0; i < 5; i++) {
+  const label = document.createElement('label');
+
+  label.innerText = replaceFirstLetter(data[i]) + ':';
+
+  if (i === 2) {
+    const select = document.createElement('select');
+    const option = document.createElement('option');
+
+    select.name = data[i];
+    select.setAttribute('data-qa', data[i]);
+
+    for (const country of offices) {
+      option.innerText = country;
+      select.append(option.cloneNode(true));
+    }
+
+    label.append(select);
+  } else {
+    const input = document.createElement('input');
+
+    input.name = data[i];
+    input.setAttribute('data-qa', data[i]);
+
+    if (data[i] === 'age' || data[i] === 'salary') {
+      input.type = 'number';
+      input.min = 0;
+    } else {
+      input.type = 'text';
+    }
+    label.append(input.cloneNode(true));
+  }
+
+  label.children[0].ariaRequired = true;
+  form.append(label.cloneNode(true));
+}
+
+form.append(button);
+
+let count = 0;
+let sortTarget = '';
+
+document.body.addEventListener('click', e => {
+  count = sortTarget.innerText !== e.target.innerText ? 1 : count + 1;
+  sortTarget = e.target;
+
+  if (count === 2) {
+    e.preventDefault();
+    count = 0;
+  }
+
+  const sort = [...employees.children].sort((a, b) => {
+    const isCompareAsString = e.target
+      .innerText === 'Name' || e.target
+      .innerText === 'Position' || e.target
+      .innerText === 'Office';
```

Comment:
And move `e.target.innerText ` to a variable
```suggestion
     const isCompareAsString =
      e.target.innerText === 'Name' ||
      e.target.innerText === 'Position' ||
      e.target.innerText === 'Office';
```